easyrest/scripts/insert_example_data.py

```diff
@@ -81,5 +83,19 @@ def fill_db(session):
 
         Rest_models.append(rest_model)
 
+    # add users
+    for myIter in range(menu_item_number):
```

Comment:
PEP8 for variable name. Anyway `iter` is bad suffix here as iterator is object we may iterate, instead of particular record. I'd use just `i` or `index` here.